src/config/laravel-exact-online.php

```diff
@@ -26,4 +26,19 @@
      * The Client country code
      */
     'exact_multi_user' => env('EXACT_MULTI_USER', false),
+
+    /**
+     * The Webhook secret
+     */
+    'exact_webhook_secret' => env('EXACT_WEBHOOK_SECRET', ''),
+
+    /**
+     * The web url
+     */
+    'exact_web_url' => env('EXACT_WEB_URL', ''),
+
+    /**
+     * The storage
+     */
+    'exact-api-storage' => env('EXACT_API_STORAGE', 'local'),
```

Comment:
Perhaps using NULL as a default value is better here, in that way the `Storage::disk` method will be called with a null value, resulting in backward compatible behaviour (it will use the default configured disk)

In that way it will not break existing applications that do not use the local disk as their default filesystem